server/api/standardized_contests.py

```diff
@@ -76,9 +76,12 @@ def process():
 
 
 def validate_standardized_contests_upload(request: Request, election: Election):
-    if election.audit_type != AuditType.BALLOT_COMPARISON:
+    if (
+        election.audit_type != AuditType.BALLOT_COMPARISON
+        and election.audit_type != AuditType.HYBRID
+    ):
         raise Conflict(
-            "Can only upload standardized contests file for ballot comparison audits."
+            "Can only upload standardized contests file for ballot comparison and hybrid audits."
```

Comment:
i actually have this change in another branch that i'm gonna put up in a PR today! but looks fine

Reply:
just kidding - i was confusing it with a similar change I made for CVR files. here's that change if you want to make it consistent: 

```py
    if election.audit_type not in [AuditType.BALLOT_COMPARISON, AuditType.HYBRID]:
        raise Conflict("Can't upload CVR file for this audit type.")
```

Reply:
I like that better, yeah.